src/SFA.DAS.Assessor.Functions/Domain/Print/Services/BlobFileTransferClient.cs

```diff
@@ -164,14 +164,14 @@ private async Task Download(string path, MemoryStream stream)
             }
         }
 
-        private CloudBlobDirectory GetCloudBlobDirectory(string path)
+        private async Task<CloudBlobDirectory> GetCloudBlobDirectory(string path)
```

Comment:
```suggestion
        private async Task<CloudBlobDirectory> GetCloudBlobDirectoryAsync(string path)
```

Reply:
The naming convention we are following is not to add Async to asynchronous methods, I believe MS does this when there is an async and non async method in the same class with the same name but we don't generally do that